### DISCUSSION TITLE
Removing custom songs

### DISCUSSION THREAD

**Discussion by paulusasol** [beat-feet/beat-feet#145] [Ideas]
It would be nice, if there was a way to remove custom songs. I at least failed to find any.

Reasons why one might want to remove a custom song might be for example:
- After adding a song, you have second songs whether the songs themes are appropriate for a light-hearted jumping game.
- The song was not loaded properly (compare #144)
- The title of the song/file name is too long to fit into a box and disturbs the view.

**Comment by pserwylo** [comment]
Absolutely agree. Have created https://github.com/beat-feet/beat-feet/issues/146 to record that this is on the todo list. This is a known issue, but I wanted to get an original version out before perfecting it.
